RULES.md

```diff
@@ -131,7 +131,7 @@ A Route color and a Route text color should be contrasting. Minimum Contrast Rat
 
 <a name="E027"/>
 
-### E027 - Missing route short name and long name
+### E027 - Missing route short name or long name
 
```

Comment:
Needs a description to say that at least one of short name or long name should be provided - both can't be blank or missing.

Also a reference to routes.txt would be nice.